### ISSUE TITLE
Explore Improvements the Event page

### ISSUE THREAD

**Issue by jeremyzilar** [GSA/digitalgov.gov#172]
We would like to explore where we could be making improvements to the event page and event listings

**Comment by jthalls** [comment]
![img_0052](https://user-images.githubusercontent.com/9658310/34419630-c435b5e4-ebb9-11e7-9c5e-235c90bdae72.jpg)
![img_0051](https://user-images.githubusercontent.com/9658310/34419632-c5c7bdd0-ebb9-11e7-862a-df20725958b0.jpg)
![img_0050](https://user-images.githubusercontent.com/9658310/34419634-c76375d0-ebb9-11e7-9c07-07793f66b3cf.jpg)

**Comment by jeremyzilar** [comment]
☝️ These are great sketches

**Comment by jeremyzilar** [comment]
Here are a few examples of a new event page:
- `online` https://federalist-proxy.app.cloud.gov/preview/gsa/digitalgov.gov/homepage-videos/event/dap-learning-series-creating-your-own-campaign-url-builder/
- `in-person` https://federalist-proxy.app.cloud.gov/preview/gsa/digitalgov.gov/homepage-videos/event/february-plain-meeting/
- `mixed` https://federalist-proxy.app.cloud.gov/preview/gsa/digitalgov.gov/homepage-videos/event/federal-challenge-prize-community-practice-quarterly-meeting-february-2018/

**Comment by thisisdano** [comment]
- I think it's confusing to see a youtube video for an event that has not yet happened. It makes me wonder whether I've missed it. or can watch it now.
- the `add to calendar` button isn't working properly for me. the little dropdown isn't appearing
- should `in person` and `online` be separate tags for mixed events? this might be a more consistent treatment if we start to add more tags
- should `online` events be `YouTube Live` instead, to more clearly communicate the venue?
- it looks like there is an em-dash `—` separating the start time and end time. this should be an en-dash `–` 

Otherwise, I like how all this is shaping up and looking. Nice and clear!

**Comment by jeremyzilar** [comment]
> I think it's confusing to see a youtube video for an event that has not yet happened. It makes me wonder whether I've missed it. or can watch it now.

Good point @thisisdano. Maybe we should hide the video area with JS till the day of the event?

**Comment by jeremyzilar** [comment]
> should online events be YouTube Live instead, to more clearly communicate the venue?

I added in the option to have multiple event types. At the moment, there are only three: `YouTube LIVE`, `In Person`, and `Online`

We can use `online` for all the events that don't happen on YouTube live.

**Comment by jeremyzilar** [comment]
The em-dash separating the start time and end time is now an en-dash

**Comment by jeremyzilar** [comment]
> the add to calendar button isn't working properly for me. the little dropdown isn't appearing
This is fixed now

**Comment by jeremyzilar** [comment]
> I think it's confusing to see a youtube video for an event that has not yet happened. It makes me wonder whether I've missed it. or can watch it now.

*Fixed* — LIVE videos get loaded on the page, but are hidden with JS until the day of the event.

**Comment by jeremyzilar** [comment]
Here is the Pull Request where the work is happening:
### Pull Request: https://github.com/GSA/digitalgov.gov/pull/450

**Comment by ToniBonittoGSA** [comment]
@jeremyzilar for "hidden with JS until the day of the event" ~ does the script need to allow for the Daylight Saving changes twice a year? Time setting was causing an issue with "Add to Calendar" button recently; https://github.com/GSA/digitalgov.gov/issues/388 & https://github.com/GSA/digitalgov.gov/pull/398
